h5p_proplist.go

```diff
@@ -40,6 +44,26 @@ func (p *PropList) Close() error {
 	return p.closeWith(h5pclose)
 }
 
+// Sets the size of the chunks used to store a chunked layout dataset.
+// https://support.hdfgroup.org/HDF5/doc/RM/RM_H5P.html#Property-SetChunk
+func (p *PropList) SetChunk(ndims uint, dim []uint) error {
+	var c_dim *C.hsize_t
+	c_dim = (*C.hsize_t)(unsafe.Pointer(&dim[0]))
+	if err := h5err(C.H5Pset_chunk(C.hid_t(p.id), C.int(ndims), c_dim)); err != nil {
+		return err
+	}
+	return nil
+}
+
+// Sets deflate (GNU gzip) compression method and compression level.
```

Comment:
please prepend `SetDeflate` to the doc string.

Reply:
Fixed